### PR TITLE
Add Cargo feature docs requirement to graduation checklist

### PR DESCRIPTION
**Pull request by sffc**
We should do this as part of stabilization.

### CODE REVIEW

**Comment by robertbastian** [comment]
Is someone going to go through all the stable components?

**Review by Manishearth — DISMISSED** [review]

**Comment by sffc** [comment]
> Is someone going to go through all the stable components?

https://github.com/unicode-org/icu4x/pull/3723

File: docs/process/graduation.md [hunk 1]
```diff
@@ -22,8 +22,10 @@ This document contains a checklist for the requirements to migrate a component f
   - [ ] Use `dep:` for enabling dependencies
 - [ ] The crate should be fully documented
   - [ ] Every exported function should have docs coverage
-  - [ ] There should be a crate-level example that illustrates a common use case for the component
-  - [ ] All options and conditional code paths should have a corresponding docs test
+  - [ ] There should be a crate-level example that illustrates a common use case for the component with the heading `# Examples`
+  - [ ] All options and conditional code paths should have a corresponding docs test with the heading `# Examples`
+  - [ ] All functions that are conditional on a Cargo feature should say so (last line before `# Examples`): ```✨ ***Enabled with the `"alloc"` Cargo feature.***```
```
Comment (robertbastian) [inline docs/process/graduation.md:27]:
```suggestion
  - [ ] All functions that are conditional on a Cargo feature should say so (last line before `# Examples`): ```✨ **Enabled with the `"alloc"` Cargo feature.**```
```

Comment (sffc) [reply docs/process/graduation.md:27]: My proposal is ***bold italic***, not just **bold**.

**Review by Manishearth — APPROVED** [review]

File: docs/process/graduation.md (hunk 1 above)
Comment (robertbastian) [reply docs/process/graduation.md:27]: Yes, my proposal is just italic

Comment (robertbastian) [reply docs/process/graduation.md:27]: Actually that's just one asterisk.

Comment (robertbastian) [reply docs/process/graduation.md:27]: I think it works well with the emoji
image

Comment (sffc) [reply docs/process/graduation.md:27]: Ok, one asterisk (italic).

Comment (robertbastian) [reply docs/process/graduation.md:27]: We might also want to drop the `"`s:
image

Comment (sffc) [reply docs/process/graduation.md]: Updated to plain italics

**Review by Manishearth — APPROVED** [review]

**Review by robertbastian — APPROVED** [review]

**Pull request merged**